examples/06-plotting/08_plot_3d_streamlines.py

```diff
@@ -0,0 +1,172 @@
+from ansys.dpf.core.plotter import DpfPlotter
+
+###############################################################################
+# Create data sources for fluids simulation result:
+fluent_files = examples.download_fluent_mixing_elbow_steady_state()
+ds_fluent = dpf.DataSources()
+ds_fluent.set_result_file_path(fluent_files["cas"][0], "cas")
+ds_fluent.add_file_path(fluent_files["dat"][1], "dat")
+
+###############################################################################
+# Create model from fluid simulation result data sources:
+m_fluent = dpf.Model(ds_fluent)
+
+###############################################################################
+# Get meshed region and velocity data
+# -----------------------------------
+# Meshed region is used as geometric base to compute the streamlines.
+# Velocity data is used to compute the streamlines. The velocity data must be nodal.
+
+# Get the meshed region:
+meshed_region = m_fluent.metadata.meshed_region
+
+# Get the velocity result at nodes:
+velocity_op = m_fluent.results.velocity()
+fc = velocity_op.outputs.fields_container()
+field = dpf.operators.averaging.to_nodal_fc(fields_container=fc).outputs.fields_container()[0]
+
+###############################################################################
+# Plot the streamlines adjusting the request
+# ------------------------------------------
+# The following steps show you how to create streamlines using DpfPlotter, with several sets
+# of parameters. It shows the issues that can happen, the adjustments that can be done.
+
+# First, a DpfPlotter is created and the streamline is created with default values.
+pl0 = DpfPlotter()
+try:
+    pl0.add_mesh(meshed_region=meshed_region, opacity=0.3)
+    pl0.add_streamlines(meshed_region=meshed_region, field=field)
+    pl0.show_figure(show_axes=True)
+except:
+    # It throws and ends here, because source point are not correctly set.
+    # They don't match with the mesh coordinates velocity values
+    # are known for.
+    # To fix this issue, the source_center parameter is used to move
+    # the streamlines source center.
+    pass
+
+# Then, using "source_center" argument that moves the source center,
+# the "return_source" arguments that displays the source, and
+# "permissive" option that allows the display of the source even if the computed
+# streamline size is zero, the source coordinates can be correctly set:
+pl1 = DpfPlotter()
+pl1.add_mesh(meshed_region=meshed_region, opacity=0.3)
+pl1.add_streamlines(
+    meshed_region=meshed_region,
+    field=field,
+    return_source=True,
+    source_center=(0.1, 0.1, 0.2),
+    permissive=True,
+)
+pl1.show_figure(show_axes=True)
+
+# After adjustment, correct values for "source_center" argument is set.
+# The "permissive" option can be removed.
+# Velocity data with small opacity to avoid to hide the streamlines can be displayed.
+# More settings are added to adapt the streamlines creation to the geometry and
+# the data of the model:
```

Comment:
```suggestion
# After the adjustment, the correct values for the "source_center" argument are set.
# You can remove the "permissive" option.
# You can display velocity data with a small opacity value to avoid hiding the streamlines.
# More settings are added to adapt the streamlines creation to the geometry and
# the data of the model:
```